services/imap/cron.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package imap
+
+import (
+	"code.gitea.io/gitea/modules/setting"
+)
+
+var (
+	c *Client
+)
+
+// FetchAllUnReadMails fetch all unread mails
+func FetchAllUnReadMails() (err error) {
```

Comment:
```suggestion
// FetchAllUnreadMails fetch all unread mails
func FetchAllUnreadMails() (err error) {
```
Of course, this will then also need to be updated for all function calls.